.github/workflows/lynx.yml

```diff
@@ -0,0 +1,58 @@
+name: '🔎 Lynx Testnet Example'
+
+on:
+  schedule:
+    - cron: "0/20 * * * *"  # Every 20 minutes
```

Comment:
This trigger will have access to GH secrets, correct?

Reply:
I think so, but GH actions docs are sometimes not great so I'm not 100% on this.